scintilla/lexilla/lexers/LexMarkdown.cxx

```diff
@@ -51,15 +54,15 @@
 #include "CharacterSet.h"
 #include "LexerModule.h"
 
-using namespace Scintilla;
+using namespace Lexilla;
 
 static inline bool IsNewline(const int ch) {
     return (ch == '\n' || ch == '\r');
```

Comment:
Correct me if I'm wrong, but it seems the lexilla sources were pulled from [the rel-5-1-1 tag](https://github.com/ScintillaOrg/lexilla/commit/d83927f9794b5213ca52fc03c094b45feb87aeda).

If that's the case, you will want to patch in [this recent fix] for LexMarkdown; the v5.1.1 release contains [a bug that could hang the editor] if a user encounters it.

[this recent fix]: https://github.com/ScintillaOrg/lexilla/commit/68ff521b53fac828d07d234d330321cfec74699d#diff-d26e1c118bcc9f873c1d3ca41945d322461a32c74c9bcee689ea0d6cb714593eR59-R65

[a bug that could hang the editor]: https://github.com/ScintillaOrg/lexilla/issues/23

cf. https://github.com/geany/geany/issues/1439#issuecomment-896544368

Reply:
I downloaded the last zip. I guess we can quickly pull 5.1.2 once out.

Reply:
No problema;  since "The Great Schism" Neil's been averaging a release per month, so it's probably just a week or two away.

Reply:
Also Neil just committed a possible fix for the dreaded Ubuntunderline DevaVu issue #2502 so picking up the next release would be really good :rocket: :champagne: